pkg/pb/metainfo.proto

```diff
@@ -24,6 +31,72 @@ service Metainfo {
     rpc ProjectInfo(ProjectInfoRequest) returns (ProjectInfoResponse);
 }
 
+message BucketItem {
+    bytes             name = 1;
+    encryption.Cipher path_cipher = 2;
+    bytes             attribution_id = 3;
+
+    google.protobuf.Timestamp created_at = 4;
```

Comment:
use gogo to have pure `time.Time` `google.protobuf.Timestamp created_at = 3 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];`